### ISSUE TITLE
Web - Page Language Changes on Page Refresh but Text Stays in Non-English Language

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#172]
@harpreetsdev @pbhatt17 @Abbychalil

Once you switch to another language like Spanish, it sets the text and page language on the web. But if you have to refresh the page, the HTML language goes back to English ( lang="en" ), but the text will stay in Spanish, French, or Malay.

There might a similar issues for iOS and Android but not likely is my guess.

Please see the attached screenshot.

![ilab - comply chain - web - page language changes if you refresh page](https://user-images.githubusercontent.com/10423487/106962812-ad05b400-670d-11eb-9c82-f8b7c353e1de.png)

**Comment by jfc3-dol** [comment]
The same still applies if a user refreshes the page after setting a non-English language that the language goes back to English even though the text is in that langauge.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abbychalil

I checked each language in Firefox, Chrome, Edge, and IE 11. For some reason, in IE 11, when I refreshed a secondary page/non-homepage, I would get a page not found 404 error message. 

So I'm assuming it's an issue with the application breaking on refresh when using IE 11 because all the other browsers worked as planned when refreshed and staying in the non-English language chosen.

I liked how I could see a quick blink from English on the page refresh, and then it switches to the chosen language—nicely, done.

So I'm going to close this issue.
